src/mixins/raster-layer-line-mixin.js

```diff
@@ -473,8 +473,14 @@ export default function rasterLayerLineMixin(_layer) {
   }
 
   _layer.viewBoxDim = createRasterLayerGetterSetter(_layer, null)
-  _layer.xDim = createRasterLayerGetterSetter(_layer, null)
-  _layer.yDim = createRasterLayerGetterSetter(_layer, null)
+
+  // linemap bbox filter gets broken if these are set for its layer, but contour
+  // needs them to be set for its bbox filter; therefore wrap in a method that
+  // can be called from contour setup prior to setting vals
+  _layer.setContourXYDims = function() {
```

Comment:
Not a huge deal, but I think it would be better if this name was not specific to the contour chart. Something generic like `initializeXYDims`

Reply:
Agree - fixed in latest commit